pmm/pmm2-api-tests.groovy

```diff
@@ -68,18 +73,15 @@ pipeline {
                 sh 'git checkout ' + env.GIT_COMMIT_HASH
             }
         }
-        stage('Setup') {
-            steps {
-                installDocker()
-                setupDockerCompose()
-            }
-        }
+
         stage('API Tests Setup')
         {
             steps{
-                withCredentials([aws(accessKeyVariable: 'AWS_ACCESS_KEY_ID', credentialsId: 'AMI/OVF', secretKeyVariable: 'AWS_SECRET_ACCESS_KEY')]) {
+                withCredentials([aws(accessKeyVariable: 'AWS_ACCESS_KEY_ID',
+                                 credentialsId: 'AMI/OVF',
+                                 secretKeyVariable: 'AWS_SECRET_ACCESS_KEY')]
+                                 ) {
                     sh '''
-                        aws ecr-public get-login-password --region us-east-1 | docker login -u AWS --password-stdin public.ecr.aws/e7j3v3n0
```

Comment:
I think we should be logging in to docker.io here to avoid the rate limit for anon.

Reply:
Yep, it's a good point